### PR TITLE
[BEAM-14140][Playground] Fix Deploy examples CI to run only on changed examples

### PR DESCRIPTION
**Pull request by AydarZaynutdinov**
[BEAM-14140]
Add condition to check if it is needed to run CI/CD step

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Format the pull request title like `[BEAM-XXX] Fixes bug in ApproximateQuantiles`, where you replace `BEAM-XXX` with the appropriate JIRA issue, if applicable. This will automatically link the pull request to the issue.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by AydarZaynutdinov** [comment]
R: @KhaninArtur @ilya-kozyrev @andreykus

File: .github/workflows/playground_deploy_examples.yml [hunk 1]
```diff
@@ -21,17 +21,48 @@ on:
...
+      - name: install deps
+        run: pip install -r requirements.txt
+        working-directory: playground/infrastructure
+      - shell: pwsh
+        name: get Difference
+        id: check_file_changed
+        run: |
+          $diff = git diff --name-only master
+          Write-Host "::set-output name=example_diff::$diff"
+      - name: has Exapmles
```
Comment (KhaninArtur) [inline .github/workflows/playground_deploy_examples.yml:53]:
```suggestion
      - name: has Examples
```

File: playground/infrastructure/checker.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+
+if __name__ == "__main__":
+  paths = ""
+  if len(sys.argv) > 1:
+    for arg in sys.argv[1:len(sys.argv) - 1]:
+      paths += arg + " "
+    paths += sys.argv[len(sys.argv) - 1]
+  else:
+    print(False)
```
Comment (KhaninArtur) [inline playground/infrastructure/checker.py:65]:
```suggestion
  paths = " ".join(sys.argv[1:])
```

File: playground/infrastructure/checker.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  print(check(paths))
```
Comment (KhaninArtur) [inline playground/infrastructure/checker.py:66]: Should we print it?
And should we run the `check` function if `paths` is empty?

**Review by KhaninArtur — CHANGES_REQUESTED** [review]

File: playground/infrastructure/checker.py (hunk 3 above)
Comment (AydarZaynutdinov) [reply playground/infrastructure/checker.py:66]: We need to print it. This code `output=$(python3 checker.py ${{ steps.check_file_changed.outputs.example_diff }})` catch all `print`s and keeps it to `output` variable.

According to my previous code if `sys.argv` doesn't contain any paths then `return False`. In another case `paths` is not empty.

Comment (KhaninArtur) [reply playground/infrastructure/checker.py:66]: Got you, fixed my first comment regarding this.

File: playground/infrastructure/checker.py (hunk 2 above)
Comment (AydarZaynutdinov) [reply playground/infrastructure/checker.py]: Done.

File: .github/workflows/playground_deploy_examples.yml (hunk 1 above)
Comment (AydarZaynutdinov) [reply .github/workflows/playground_deploy_examples.yml]: Done.

**Review by KhaninArtur — APPROVED** [review]
LGTM

**Comment by andreykus** [comment]
LGTM

**Comment by AydarZaynutdinov** [comment]
R: @pabloem

**Comment by pabloem** [comment]
LGTM thanks all!

**Pull request merged**